.github/workflows/inventory.yml

```diff
@@ -14,25 +14,52 @@ jobs:
     name: Update Go Inventory
     runs-on: pub-hk-ubuntu-22.04-small
     steps:
+      - uses: heroku/use-app-token-action@main
+        id: generate-token
+        with:
+          app_id: ${{ vars.LINGUIST_GH_APP_ID }}
+          private_key: ${{ secrets.LINGUIST_GH_PRIVATE_KEY }}
+
       - name: Checkout Repo
         uses: actions/checkout@v3
+
       - id: install-rust-toolchain
         name: Install Rust Toolchain
         uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
-      - id: set-diff-msg
-        name: Set Diff Message
-        run: echo "DIFF_MSG=$(cargo run --bin diff_inventory inventory.toml)" >> $GITHUB_OUTPUT
+
+      - name: Set Diff Message
+        id: set-diff-msg
+        run: |
+          delimiter="$(openssl rand -hex 8)"
+          {
+            echo "msg<<${delimiter}"
```

Comment:
The syntax seemed strange to me, but I see this is actually the documented way for multi-line strings in `GITHUB_OUTPUT` - TIL :-)
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

Reply:
Yeah, weird indeed. Copied from https://github.com/heroku/buildpacks-nodejs/blob/main/.github/workflows/_update-inventory.yml#L47-L55. Glad to see this syntax is documented tho.